### ISSUE TITLE
js layout on images loaded

### ISSUE THREAD

**Issue by mikeyrayvon** [interglobalvision/pichon-mx#34]
layout isnt waiting for images to be loaded so it ends up all wrong sometimes? difficult to reproduce

**Comment by varffs** [comment]
Yeah its really weird. I'm wondering if imagesLoaded doesnt work with src-sets or something?

**Comment by mikeyrayvon** [comment]
coud b
